.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```

Comment:
It seems to be a personal configuration. 
Can we remove it from the PR?

Reply:
done